docs/start/authentication/graphql-storefront.mdx

```diff
@@ -94,8 +97,9 @@ Accept: application/json
 Content-Type: application/json
 
 {
-  "channel_ids": [1, 2, 3] // array of integers (must be a valid channel IDs on the store)
-  "expires_at": 1602288000 // when the token will expire, as an integer unix timestamp (in seconds)
+  "channel_ids": [1, 2, 3], // array of integers (must be a valid channel IDs on the store)
+  "expires_at": 1602288000, // when the token will expire, as an integer unix timestamp (in seconds)
+  "dev_mode": true
```

Comment:
same as above

Reply:
Thanks, good idea, just added a section. I reworded it since currently, it's just the login & logout mutations that don't work in development mode